cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,5 +1,5 @@
 {% set heapster_memory = "300Mi" -%}
-{% if pillar['num_nodes'] is defined -%}
+{% if pillar['num_nodes'] is defined and pillar['num_nodes'] is not None -%}
```

Comment:
You could just shorthand this to `pillar.get('num_nodes') is not None`, couldn't you?